### DISCUSSION TITLE
Tiles images and or icons on Start screen

### DISCUSSION THREAD

**Discussion by MrCoolAndroid** [DrewNaylor/Retiled#172] [Ideas]
I found your project on GitHub not too long ago and I really like it! I see you use some C# for the start screen, I could help with the tiles but I'm not sure if it is only C# or other languages

**Comment by DrewNaylor** [comment]
Thanks! Actually, I'm currently using QML with Python and the C# is archived code, with some VB.NET that I need for another project and haven't fully deleted yet.

**Comment by MrCoolAndroid** [comment]
Ooh okay, I have little knowledge on Python heh, still very good project, it will be amazing when it's done!El 23 ene. 2023 0:17, Drew Naylor ***@***.***> escribió:
Thanks! Actually, I'm currently using QML with Python and the C# is archived code, with some VB.NET that I need for another project and haven't fully deleted yet.

**Reply by DrewNaylor** [reply]
Thank you for the offer, anyway!
